### PR TITLE
Misc. documentation and source comment typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L ,ang,ba,childs,eiter,forse,objext,pevent,shouldbe,thru,uint,unselect,vertexes --skip="./synfig-studio/po,./synfig-core/po,/synfig-core/m4,./synfig-osx/,./gtkmm-osx,./bugs,.*.eps,*.sif"`

### CODE REVIEW

File: synfig-core/src/modules/mod_geometry/advanced_outline.cpp [hunk 1]
```diff
@@ -805,12 +805,12 @@ Advanced_Outline::sync_vfunc()
...
-					// This modification fixes bad render when first widht
-					// point is not interpolate after and next widhtpoint is
-					//  interpolate before. Noticiable for the FLAT case
+					// This modification fixes bad render when first width
+					// point is not interpolate after and next widthpoint is
+					//  interpolate before. Noticeable for the FLAT case
 					// or when the width is smaller than the step on the bezier.
 					ipos=ipos+EPSILON;
-					// If we have just done a tip width side tipe after not interpolate
+					// If we have just done a tip width side type after not interpolate
```
Comment (luzpaz) [inline synfig-core/src/modules/mod_geometry/advanced_outline.cpp]: please review

File: synfig-core/src/synfig/rendering/primitive/contour.h [hunk 2]
```diff
@@ -137,7 +137,7 @@ class Contour: public etl::shared_object
 	void add_chunks(const ChunkList &chunks)
 		{ if (!chunks.empty()) add_chunks(&chunks.front(), &chunks.back() + 1); }
 
-	//! attache list from (end-1) to begin
+	//! attached list from (end-1) to begin
```
Comment (luzpaz) [inline synfig-core/src/synfig/rendering/primitive/contour.h]: should this be `attach` or `attached` ?

File: synfig-core/src/synfig/time.cpp [hunk 3]
```diff
@@ -326,7 +326,7 @@ Time::get_string(float fps, Time::Format format)const
 Time
 Time::round(float fps)const
 {
-	// the aim is to make results for the same frame are absolutelly idential
+	// the aim is to make results for the same frame absolutely identical
```
Comment (luzpaz) [inline synfig-core/src/synfig/time.cpp]: Should this be `the aim is to make sure the results for the same frame are absolutely identical` ?

**Comment by morevnaproject** [comment]
Thank you for the fixes! I taking them for review. ^__^

**Comment by ankit-kumar-dwivedi** [comment]
@morevnaproject is ETL still maintained? I saw in this PR there are edits in ETL files too

File: synfig-core/src/synfig/rendering/primitive/contour.h (hunk 2 above)
Comment (ice0) [reply synfig-core/src/synfig/rendering/primitive/contour.h]: attach

**Comment by ice0** [comment]
> @morevnaproject is ETL still maintained? I saw in this PR there are edits in ETL files too

@ankit-kumar-dwivedi yep, it still maintained.

File: synfig-core/src/synfig/rendering/primitive/contour.h (hunk 2 above)
Comment (luzpaz) [reply synfig-core/src/synfig/rendering/primitive/contour.h]: c6d7fec

**Comment by ice0** [comment]
@luzpaz First of all, thanks for your awesome work!

All is looks nice and ready to merge. But you also need to fix .po files for each correction that starts with _(".
I think it is better to split this work into two PRs.
1. Remove all changes which begins with _(", and i will merge them.
2. Create new PR with that changes and fixed .po files.

**Comment by morevnaproject** [comment]
@ice0 Thanks for taking time on review! Just want to mention that with .po files it's not that simple. If @luzpaz will fix them now, then they will get overwritten.

This is because we use following schema:
Build system generates .pot (PO template) file from source code. Then it is pushed (manually) to Transifex service (https://www.transifex.com/morevnaproject/synfig/). Translators translate strings there and translations are stored as .po files. Those files are fetched to synfig source (manually, using script).

So, first we have to lock translations on Transifex (to avoid someone doing changes in parallel with us) fetch latest .po files to Synfig source, make corrections, push .po files back to Transifex, unlock translations. ^__^

**Comment by luzpaz** [comment]
I always ignore `.po files for the exact reason @morevnaproject mentioned. How do you want me to proceed ?

File: synfig-core/src/modules/lyr_std/sphere_distort.cpp [hunk 4]
```diff
@@ -163,7 +163,7 @@ Layer_SphereDistort::get_param_vocab()const
 
 	ret.push_back(ParamDesc("clip")
 		.set_local_name(_("Clip"))
-		.set_description(_("When cheked, the area outside the Radius are not distorted"))
+		.set_description(_("When checked, the area outside the Radius are not distorted"))
```
Comment (ice0) [inline synfig-core/src/modules/lyr_std/sphere_distort.cpp]: need to revert back

File: synfig-core/src/modules/lyr_std/timeloop.cpp [hunk 5]
```diff
@@ -148,7 +148,7 @@ Layer_TimeLoop::get_param_vocab()const
 
 	ret.push_back(ParamDesc("duration")
 		.set_local_name(_("Duration"))
-		.set_description(_("Lenght of the loop"))
+		.set_description(_("Length of the loop"))
```
Comment (ice0) [inline synfig-core/src/modules/lyr_std/timeloop.cpp]: need to revert back

File: synfig-core/src/modules/mod_png/trgt_png_spritesheet.cpp [hunk 6]
```diff
@@ -178,7 +178,7 @@ png_trgt_spritesheet::set_rend_desc(RendDesc *given_desc)
 
 	if (sheet_width * sheet_height > 5000 * 2000)
 	{
-		synfig::error(strprintf(_("The image is too large. It's size must be not more than 5000*2000=10000000 px. Now is %d*%d=%d px."), 
+		synfig::error(strprintf(_("The image is too large. It's size must be not more than 5000*2000=10000000 px. Currently it's %d*%d=%d px."),
```
Comment (ice0) [inline synfig-core/src/modules/mod_png/trgt_png_spritesheet.cpp]: need to revert back

File: synfig-studio/src/gui/app.cpp [hunk 7]
```diff
@@ -3901,7 +3901,7 @@ App::open_from_temporary_filesystem(std::string temporary_filename)
 			one_moment.hide();
 
 			if(instance->is_updated() && App::dialog_message_2b(
-				_("Newer version of this file avaliable on the CVS repository!"),
+				_("Newer version of this file available on the CVS repository!"),
```
Comment (ice0) [inline synfig-studio/src/gui/app.cpp]: need to revert back

File: synfig-studio/src/gui/instance.cpp [hunk 8]
```diff
@@ -206,7 +206,7 @@ studio::Instance::run_plugin(std::string plugin_path)
 {
 	handle<synfigapp::UIInterface> uim = this->find_canvas_view(this->get_canvas())->get_ui_interface();
 
-	string message = strprintf(_("Do you realy want to run plugin for file \"%s\"?" ),
+	string message = strprintf(_("Do you really want to run plugin for file \"%s\"?" ),
```
Comment (ice0) [inline synfig-studio/src/gui/instance.cpp]: need to revert back

File: synfig-core/src/modules/lyr_std/sphere_distort.cpp (hunk 4 above)
Comment (morevnaproject) [reply synfig-core/src/modules/lyr_std/sphere_distort.cpp]: Why?

File: synfig-studio/src/gui/instance.cpp (hunk 8 above)
Comment (morevnaproject) [reply synfig-studio/src/gui/instance.cpp]: @ice0 Also, why? ^__^

File: synfig-core/src/modules/lyr_std/sphere_distort.cpp (hunk 4 above)
Comment (ice0) [reply synfig-core/src/modules/lyr_std/sphere_distort.cpp]: I think it will be easier to fix .po files if this changes will be in a separate commit.

Comment (morevnaproject) [reply synfig-core/src/modules/lyr_std/sphere_distort.cpp]: Ah, I see. No, that doesn't makes any influence. ^__^

File: synfig-core/src/synfig/time.cpp (hunk 3 above)
Comment (morevnaproject) [reply synfig-core/src/synfig/time.cpp]: It's ok to leave as "the aim is to make results for the same frame absolutely identical" ^__^

**Comment by morevnaproject** [comment]
Looks good to me! Thank you! ^__^

**Comment by morevnaproject** [comment]
@luzpaz I have locked translations on Transifex and added latest .po files to Synfig repository - de8c0ee446c8416e170ae97b1cee0d1e72c5bf58.
Now, it is fine to fix .po files. I have created a separate issue for this task - #835

**Comment by luzpaz** [comment]
Will try to get to this as soon as I can.

**Comment by morevnaproject** [comment]
Okay, #836 and #837 are merged now. 
Strings are pushed back to Transifex.
Transifex now unlocked.

The issue is closed. Much thanks to everyone who participated! ^__^

**Pull request merged**